examples/bpi-monolith-test-suite/bpi/components/remotes/privateInput.ts

```diff
@@ -0,0 +1,4 @@
+export interface privateInput {
```

Comment:
nit: prefix privateInput interface with `I`, `IPrivateInput`
or just `PrivateInput`

Reply:
This file removed as unused code